src/js/components/UserTimeline.js

```diff
@@ -32,11 +36,22 @@ class UserTimeline extends React.Component {
         }
     }
 
+    displayModal(toggle, content){
+        let modalContent;
+        if(content) {
```

Comment:
you can optimize as this conditional doesn't do anything.  If the content is not passed in, the content variable is 'undefined', the same as modalContent

Reply:
btw, what is content/modelContent represent?

Reply:
Content/modalContent represents the tweet to be replied to. I've renamed it to "tweet" as it seems more fitting